vllm/_custom_ops.py

```diff
@@ -541,11 +547,34 @@ def cutlass_scaled_mm(a: torch.Tensor,
     n = b.shape[1]
 
     if current_platform.is_rocm():
-        triton_scaled_mm_module = importlib.import_module(
-            "vllm.model_executor.layers.quantization.compressed_tensors."
-            "triton_scaled_mm")
-        triton_scaled_mm = triton_scaled_mm_module.triton_scaled_mm
-        return triton_scaled_mm(a, b, scale_a, scale_b, out_dtype, bias)
+        if is_rocm_aiter_gemm_w8a8_scaled_mm_enabled():
```

Comment:
This shouldn't live inside `cutlass_scaled_mm`, has nothing to do with cutlass. This code should just live inside `AiterScaledMMLinearKernel.apply`.

I know the Triton kernel is here but it shouldn't be either, I'm currently refactoring that.